src/commands/skill-templates/gauntlet-run-extract-prompt.md

```
@@ -0,0 +1,121 @@
+3. Read it and find all lines containing `[FAIL]`
+4. For each `[FAIL]` line, extract the referenced file path
+5. Read each referenced file:
+   - **`.log` files** (check gates): Extract error output. Look for `--- Fix Instructions ---` sections and `--- Fix Skill: <name> ---` sections. Include their full content.
+   - **`.json` files** (review gates): Parse the JSON. Find violations where `status` is `"new"`. For each, extract: `file`, `line`, `issue`, `priority`, `fix`.
+
+## Output Format
+
+Return a plain-text summary using EXACTLY this format:
+
+For check failures:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)